echo-telemetry/echo-telemetry.gradle

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+  implementation project(':echo-model')
+  implementation project(':echo-notifications')
+  implementation project(':echo-proto')
+  implementation 'com.squareup.retrofit:retrofit'
+  implementation 'com.squareup.retrofit:converter-jackson'
+  implementation 'com.netflix.spinnaker.kork:kork-web'
+  implementation 'com.google.protobuf:protobuf-java-util:3.7.1'
```

Comment:
Would like to see this dep get extracted out to `kork/spinnaker-dependencies`

Reply:
The protobuf bom is in spinnaker-deps, and version has been removed here.